include/openssl/e_os2.h

```diff
@@ -276,6 +276,17 @@ typedef unsigned __int64 uint64_t;
 #  endif
 # endif
 
+/* Format specifier for printing size_t */
+# if (defined(__STDC_VERSION__) && __STDC_VERSION__ >= 199901L)
```

Comment:
I'm getting confused. VMS has a C99 compiler that doesn't support %zu, so you switch to C94 (which doesn't exist?) instead?

Anyway this is implemented by libc, not the compiler. So even if your compiler doesn't support %zu your libc might.